### ISSUE TITLE
devtools protocol: does not trigger change event in select element

### ISSUE THREAD

**Issue by takeya0x86** [webdriverio/webdriverio#5379]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 6.1.9
 - **Mode:**  WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:**  sync
 - **Node.js version:**  12.16.3
 - **NPM version:**  6.14.4
 - **Browser name and version:** Chrome 81
 - **Platform name and version:** macOS 10.15.4
 - **Additional wdio packages used (if applicable):** 

**Config of WebdriverIO**
`automationProtocol: 'devtools'`

**Describe the bug**
https://takeya0x86.github.io/playground/0509_change_event.html
This page contains a select element and when select option, selected value displays below area.
It is implemented by Javascript `change` event.
But using webdriverio devtools protocol, `change` event doesn't triggered.

**To Reproduce**
```javascript
it('Select and check', () => {
  browser.url('https://takeya0x86.github.io/playground/0509_change_event.html');
  expect($('#selected')).toHaveText('-');

  $('#select1').selectByVisibleText('Option B');
    
  expect($('#select1 > option[value="B"]')).toBeSelected();
  expect($('#selected')).toHaveText('B'); // => fail!!
});
```

**Expected behavior**
trigger `change` event

**Log**
https://gist.github.com/takeya0x86/969a090370586a15905197128ee662d4

**Additional context**
When `automationProtocol: 'webdriver'`, it works fine.
https://gist.github.com/takeya0x86/ecb44650f613ee8b6226ec4a43a34d8e

**Comment by leobastiani** [comment]
Same issue here!

**Comment by christian-bromann** [comment]
Fixed in #6164
